sympy/functions/elementary/trigonometric.py

```diff
@@ -483,6 +483,10 @@ def _eval_rewrite_as_besselj(self, arg, **kwargs):
         from sympy.functions.special.bessel import besselj
         return sqrt(pi*arg/2)*besselj(S.Half, arg)
 
+    def  _eval_rewrite_as_hyper(self, arg, **kwargs):
+        from sympy.integrals.meijerint import _rewrite_single
+        return _rewrite_single(sin(arg), arg)[0][0][2]*sqrt(pi)
```

Comment:
If you look at what `_rewrite_single` does then you can see that the factor of `sqrt(pi)` is already computed there:
```python
In [1]: from sympy.integrals.meijerint import _rewrite_single

In [2]: _rewrite_single(sin(x), x)
Out[2]: ([(sqrt(pi), 0, meijerg(((), ()), ((1/2,), (0,)), x**2/4))], True)
```
It should be reused rather than hardcoding `*sqrt(pi)` here.

Reply:
I am not sure this relation is valid for all x. We need `-pi/2 < arg(x) <= pi/2` according to Wikipedia. 

More checking is needed to see if this is valid for all `x` including negative, positive, zero, imaginary different quadrants of the complex plane etc.

Reply:
> If you look at what _rewrite_single does then you can see that the factor of sqrt(pi) is already computed there:

If we reuse it, the final result may resemble something like this:
`sin(x).rewrite(hyper)[0][0][2]*sin(x).rewrite(hyper)[0][0][0]`

Reply:
> I am not sure this relation is valid for all x. We need `-pi/2 < arg(x) <= pi/2` according to Wikipedia.
> 
> More checking is needed to see if this is valid for all `x` including negative, positive, zero, imaginary different quadrants of the complex plane etc.

We can add more test cases. Can you give me examples of how to write test cases for complex numbers and imaginary numbers in different planes?